netdata-installer.sh

```diff
@@ -806,8 +818,10 @@ install_go() {
 		run rm -rf "${NETDATA_STOCK_CONFIG_DIR}/go.d"
 		run rm -rf "${NETDATA_STOCK_CONFIG_DIR}/go.d.conf"
 		run tar -xf "${tmp}/config.tar.gz" -C "${NETDATA_STOCK_CONFIG_DIR}/"
+		run chown -R "${ROOT_USER}:${NETDATA_GROUP}" "${NETDATA_STOCK_CONFIG_DIR}"
 
 		run mv "${tmp}/$GO_PACKAGE_BASENAME" "${NETDATA_PREFIX}/usr/libexec/netdata/plugins.d/go.d.plugin"
+		run chown root:${NETDATA_GROUP} "${NETDATA_PREFIX}/usr/libexec/netdata/plugins.d/apps.plugin"
 		run chmod 4750 "${NETDATA_PREFIX}/usr/libexec/netdata/plugins.d/go.d.plugin"
```

Comment:
Why? Everything else is 4750

Reply:
No

```
-rwxr-xr-x 1 root root         154 янв 19 00:22 alarm-email.sh
-rwxr-xr-x 1 root root       78908 янв 19 00:22 alarm-notify.sh
-rwxr-xr-x 1 root root         301 янв 19 00:22 alarm-test.sh
-rwxr-x--- 1 root netdata   100848 янв 19 00:22 apps.plugin
-rwxr-xr-x 1 root root        5408 янв 19 00:22 cgroup-name.sh
-rwsr-x--- 1 root netdata    43336 янв 19 00:22 cgroup-network
-r-xr-x--- 1 root root        6931 янв 19 00:22 cgroup-network-helper.sh
-rwxr-xr-x 1 root root        1259 янв 19 00:22 charts.d.dryrun-helper.sh
-rwxr-xr-x 1 root root       19147 янв 19 00:22 charts.d.plugin
-rwxr-xr-x 1 root root        5540 янв 19 00:22 fping.plugin
-rwsr-x--- 1 root root    12620747 янв 19 00:22 go.d.plugin
-rwxr-xr-x 1 root root        8145 янв 19 00:22 health-cmdapi-test.sh
-rw-r--r-- 1 root root        6631 янв 19 00:22 loopsleepms.sh.inc
-rwxr-xr-x 1 root root       11084 янв 19 00:22 node.d.plugin
-rwxr-xr-x 1 root root       14827 янв 19 00:22 python.d.plugin
-rwxr-xr-x 1 root root        7287 янв 19 00:22 tc-qos-helper.sh
```

Reply:
```
2019-01-19 00:34:28: netdata INFO  : PLUGINSD[go.d] : thread created with task id 20088
2019-01-19 00:34:28: netdata INFO  : PLUGINSD[go.d] : connected to '/opt/netdata/usr/libexec/netdata/plugins.d/go.d.plugin' running on pid 20090
sh: /opt/netdata/usr/libexec/netdata/plugins.d/go.d.plugin: Permission denied
sh: line 0: exec: /opt/netdata/usr/libexec/netdata/plugins.d/go.d.plugin: cannot execute: Permission denied
2019-01-19 00:34:28: netdata ERROR : PLUGINSD[go.d] : read failed
2019-01-19 00:34:28: netdata ERROR : PLUGINSD[go.d] : '/opt/netdata/usr/libexec/netdata/plugins.d/go.d.plugin' (pid 20090) disconnected after 0 successful data collections (ENDs).
2019-01-19 00:34:28: netdata ERROR : PLUGINSD[go.d] : child pid 20090 exited with code 126.
2019-01-19 00:34:28: netdata ERROR : PLUGINSD[go.d] : '/opt/netdata/usr/libexec/netdata/plugins.d/go.d.plugin' (pid 20090) exited with error code 126. Disabling it.
2019-01-19 00:34:28: netdata INFO  : PLUGINSD[go.d] : thread with task id 20088 finished
```

Reply:
actually why all plugins not `root:netdata` ?

Reply:
I have no idea.

Reply:
I just modeled permissions after `apps.plugin`.

I don't want to touch anything in installer script since this whole thing is very fragile. A lot of stuff shouldn't be here and should be in a build system, but during development something took a wrong turn and whole script become a build system by itself.